CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
```

Comment:
I would put a link here which would show a diff between the latest release and new commits:
For example, if `1.0.0` is the latest release: 
`https://github.com/pusher/push-notifications-server-java/compare/1.0.0...HEAD`

Reply:
Ok, I'll do it outside this PR!